indico/web/templates/message_box.html

```diff
@@ -1,5 +1,6 @@
-{% macro message_box(type, message=none, fixed_width=false, large_icon=false, footer=none) %}
-    <div class="{{ type }}-message-box {%- if fixed_width %} fixed-width{% endif %}{%- if large_icon %} large-icon{% endif %}"
+{% macro message_box(type, message=none, fixed_width=false, large_icon=false, footer=none, classes='') %}
+    <div class="{{ type }}-message-box {%- if fixed_width %} fixed-width{% endif %}{%- if large_icon %} large-icon{% endif %}
+                {{ classes }}"
          {{ kwargs | html_params }}>
```

Comment:
i'd put this in the previous line now that we have a new line with lots of space for it